cpp/src/barretenberg/stdlib/primitives/field/field.test.cpp

```diff
@@ -947,6 +948,269 @@ template <typename Composer> class stdlib_field : public testing::Test {
         EXPECT_EQ(composer.failed(), true);
         EXPECT_EQ(composer.err(), "field_t::pow exponent accumulator incorrect");
     };
+
+    static void test_array_length()
+    {
+        Composer composer = Composer();
+
+        constexpr size_t ARRAY_LEN = 10;
+        std::array<fr, ARRAY_LEN> values;
+        std::array<field_ct, ARRAY_LEN> values_ct;
+
+        constexpr size_t filled = 6;
```

Comment:
super nitpick: If `ARRAY_LEN` Is capitalized, shouldn't `filled` be? :D

Reply:
Maybe. `ARRAY_LEN` is caps because our stack lengths that we use in a3 are caps (constants). So didn't really think `filled` should be caps.